### ISSUE TITLE
Code Review

### ISSUE THREAD

**Issue by PiotrTe** [Kreytal/U_W#2]
Good C# code commenting practices are present in the code. Clear comments make it easier to understand the code.

There are however some issues with the ruffle shuffling algorithm 
https://github.com/Kreytal/U_W/blob/fdbae2728bceba9ea9fc93cbb14c77728e67dec6/Pack.cs#L50-L93

First of them is that riffle shuffle is performed only once which is not enough to thoroughly shuffle the deck using this method, some websites recommend at least 5-7 shuffles before cards are ready to play

Another one is this method, which kind of defeats the whole purpose of building your own, simple shuffle algorithm, right at the beginning there is a build-in method for shuffling and after that the rest of the algorithm plays out
https://github.com/Kreytal/U_W/blob/fdbae2728bceba9ea9fc93cbb14c77728e67dec6/Pack.cs#L53-L55

I believe that the previous code was inserted to mitigate this error in the code, code splits the stack in 2 and then grabs left, then right pile every time, instead of it being random it is predictable and always gives the same outcome
https://github.com/Kreytal/U_W/blob/fdbae2728bceba9ea9fc93cbb14c77728e67dec6/Pack.cs#L64-L74

Another issue I was able to find is that Deal method is always returning the first number in the list of dealt cards, and if an app mentioned in the brief required to deal 1 cards few consecutive times, the card returned by a value would always be the same card
https://github.com/Kreytal/U_W/blob/fdbae2728bceba9ea9fc93cbb14c77728e67dec6/Pack.cs#L99-L102
but in the next function to deal multiple cards, function iterates though whole list and shows all cards, if the function were to be called again it would return all cards including those that already are dealt to the player
https://github.com/Kreytal/U_W/blob/fdbae2728bceba9ea9fc93cbb14c77728e67dec6/Pack.cs#L119-L123

Validation method is very robust and it was able to detect all of the improper values I tested with it, although by looking at the code it would have been beneficial to add ConsoleWriteline to the  Validation function as it is already taking input so might as well send the instruction for the input when taking it. Or just leave it blank when its not needed
https://github.com/Kreytal/U_W/blob/fdbae2728bceba9ea9fc93cbb14c77728e67dec6/Testing.cs#L21-L22

**Comment by Kreytal** [comment]
Thanks for your review. Thank you that you have spotted all of these issues, especially the second type shuffle, completely got lost when I was doing it.

-1.st Deal fixed.
-Included message before taking input for the user.
-Made randomized riffle shuffle.
-Made the riffle shuffle repeat 5 to 7 times.
